packages/material-ui/src/Portal/index.js

```diff
@@ -1 +1,3 @@
-export { default } from './Portal';
+import { Portal } from '@material-ui/unstyled';
+
+export default Portal;
```

Comment:
I think that we could blindly reexport everything.

Reply:
Tried to but the problem is with a compilation

When I try 

`export * from '@material-ui/unstyled/Portal';`

![image](https://user-images.githubusercontent.com/8037810/107871952-efb65180-6eae-11eb-9544-f6712c852f4c.png)

Reply:
I have seen this error before, with the slider. Having a second look.

Reply:
I think the problem is with the rollup config with nodeResolve, I had a similar problem with webpack where you must properly resolve externals which are externals.

Reply:
Ok, took me one hour, but time well spent 😁, It's important to have a low bundle size overhead when using a single module. It's a significant fear developers have for adopting the library; we need to keep it best in class

Reply:
cc @mnajdova, the build is green 🎉  It took us 3 iterations, but it seems that unstyled starts to deliver.

Reply:
Glad to see another component moved, good job!